houssam98/IVT-FaceRecognition-Testing
# Added files to UI test (#13)

* Add files via upload

Test Log File

* Add files via upload

KeyWord Testing approach file

* Add files via upload

* Add files via upload

* Add files via upload

* Add files via upload

ClientStory-ManualTest

* Add files via upload

codes and Scripts

* Add files via upload

capture-play-verify

* Add files via upload

SmokeTest

* Add files via upload

CrossBrowsersTest

* Add files via upload

'seleniumLibrary.robot' missing file, must install to work with the robot framework.
